src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -245,6 +256,21 @@ public AzureAppConfigurationOptions TrimKeyPrefix(string prefix)
             return this;
         }
 
+        /// <summary>
+        /// Trims the provided prefix from the keys of all feature flags retrieved from Azure App Configuration.
+        /// </summary>
+        /// <param name="prefix">The prefix to be trimmed from all feature flags retrieved from Azure App Configuration.</param>
+        public AzureAppConfigurationOptions TrimFeatureFlagPrefix(string prefix)
```

Comment:
>TrimFeatureFlagPrefix [](start = 44, length = 21)

What happens when a user configures both `TrimKeyPrefix` and `TrimFeatureFlagPrefix`? It looks like the `TrimKeyPrefix` would apply to the trimmed feature flags too.

Reply:
`TrimKeyPrefix` comes into play after all adapters have been processed on a key-value, whereas `TrimFeatureFlagPrefix` would try to trim the prefix from feature flag IDs while feature management adapter is processing the key-value.
For example, if I have these key-values and feature flags:
- test_keyvalue1
- .appconfig.featureflag/test_feature1 

`TrimKeyPrefix("test_")` would trim the prefix from the key-value, not the feature flag, because the ***processed*** feature flag key doesn't start with the prefix. Your config would have two keys: `keyvalue1` and `FeatureManagement:test_feature1`.

On the other hand, `TrimFeatureFlagPrefix("test_")` would trim the prefix from feature flag ID. That's how you could get `FeatureManagement:feature1` in your config.